LayoutTests/platform/ios/TestExpectations

```diff
@@ -1103,6 +1103,7 @@ fast/media/video-element-in-details-collapse.html [ Skip ]
 fast/frames/user-gesture-timestamp-propagation.html [ Failure ]
 fast/events/mouse-click-different-mouseDown-mouseUp-nodes.html [ Skip ]
 fast/scrolling/scrolling-inside-scrolled-overflowarea.html [ Skip ]
+fast/events/mouse-events-on-textarea-resize.html [ Skip ]
```

Comment:
nit: you might want to put this at the end of the `# CSS resizers aren't supported on iOS` section, since the textarea resizer is a CSS resizer.

Reply:
Done.